Main/mail.py

```diff
@@ -0,0 +1,83 @@
+import smtplib
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+from email.mime.base import MIMEBase
+from email import encoders
+
+# Mail sent to : treemusketeers32@gmail.com
+# Password : Dhootkilillyput
+# Mail sent from : spyeyemusk42@gmail.com
+# Password : Dhootkilillyput
+
+# Setup port number and server name
+
+smtp_port = 587                 # Standard secure SMTP port
+smtp_server = "smtp.gmail.com"  # Google SMTP Server
+
+# Set up the email lists
+email_from = "spyeyemusk42@gmail.com"
+
+
+# Define the password (better to reference externally)
+pswd = "dosi juxf dbjx mgeb" # As shown in the video this password is now dead, left in as example only
+
+
+# name the email subject
+subject = "Intruder Alert"
+
+
+
+# Define the email function (dont call it email!)
+def send_emails(path):
+    email_list = ["treemusketeers32@gmail.com"]
```

Comment:
You can add the college email ids of our team members as well here.

Reply:
cool